models_config.json

```diff
@@ -0,0 +1,25 @@
+[
+  {
+    "provider_name": "openai",
+    "api_format": "openai",
+    "llm_model_ids": ["gpt-4", "gpt-3-5-turbo"],
+    "embedding_model_ids": ["text-embedding-3-small"],
+    "api_key_env_var": "OPENAI_API_KEY"
```

Comment:
Not able to find, where do we set API keys? Either in the Gateway Class, initialize these vars from settings.py?

Reply:
Its set in the gateway class - check the get_embedder and get_llm function